network/prober/prober_test.go

```diff
@@ -264,6 +264,54 @@ func TestAsyncMultiple(t *testing.T) {
 	}
 }
 
+func TestWithPathOption(t *testing.T) {
+	path := "/correct/probe/path/"
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		t.Logf("Want: %s, Got: %s\n", path, r.URL.Path)
+		if r.URL.Path != path {
+			w.WriteHeader(404)
+		}
+	}))
+	defer ts.Close()
+
+	tests := []struct {
+		name    string
+		options []interface{}
+		success bool
+		expErr  bool
+	}{{
+		name:    "no path",
+		options: []interface{}{ExpectsStatusCodes([]int{http.StatusOK})},
```

Comment:
you can expect `404` and then get rid of `success` variable altogether?

Reply:
That's nice.  Done